Dockerfile

```diff
@@ -247,6 +293,23 @@ COPY --from=precompiler /opt/mastodon/public/packs /opt/mastodon/public/packs
 COPY --from=precompiler /opt/mastodon/public/assets /opt/mastodon/public/assets
 # Copy bundler components to layer
 COPY --from=bundler /usr/local/bundle/ /usr/local/bundle/
+# Copy libvips components to layer
+COPY --from=build /usr/local/bin/vips* /usr/local/bin
+COPY --from=build /usr/local/lib/libvips* /usr/local/lib
+COPY --from=build /usr/local/lib/pkgconfig/vips* /usr/local/lib/pkgconfig
+
+RUN \
+# Symlink libvips components
+  ln -sf /usr/local/lib/libvips.so.42.17.2 /usr/local/lib/libvips.so.42; \
```

Comment:
I suspect those ABI versions will change when libvips with libvips update. Can we find another way to make them work?

I see two different ways:
- Use the trick described here to use a plain `cp` command, mounting the layer for the command: https://github.com/moby/moby/issues/40449
- Use a custom prefix when building vips (like `/usr/local/vips`), then copy whole directories from it, as symlinks are copied as-it when the source is a directory. This could allow to copy every vips-related file at once.